docs/tools/bench.md

```diff
@@ -16,6 +16,7 @@ The current benchmarks are:
 * `sieve`: Print all prime numbers up to *n* using the [Sieve of Eratosthenes][sievee].
 * `collatz`: Print the [Collatz][collatz] sequence starting at *n*. Note: it is not known whether this will terminate for all *n*.
 * `ackermann`: Print the value of `Ack(m, n)`, the [two-argument Ackermann-Péter function][ackermann].
+* `sum-bit`: Print the number of bits in the binary representation of the input integer.
```

Comment:
Maybe this should be "the number of 1 bits"? It's not the totally length of the binary representation (i.e., the position of the highest 1).